src/components/MainView/MainView.jsx

```diff
@@ -3,19 +3,23 @@ import { MovieCard } from "../MovieCard/MovieCard";
 import { MovieView } from "../MovieView/MovieView";
 import { LoginView } from "../LoginView/LoginView";
 import { SignupView } from "../SignUpView/SignUpView";
+import { NavigationBar } from "../NavigationBar/NavigationBar";
+import { ProfileView } from "../ProfileView/ProfileView";
+import Row from "react-bootstrap/Row";
```

Comment:
Make use of distructaring 

import {Row, Col} from 'react-bootstrap';

Reply:
Thank you. Changed.

Reply:
You have conflicts

Reply:
I mean Branch conflict with the target branch
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/33962360/231423687-380b27cc-3fc4-4794-b7f4-b7689219257e.png">